components/portal/timeout/controllers.js

```diff
@@ -31,6 +31,8 @@ define(['angular'], function(angular) {
   ) {
     var checkInactiveDelay = 1; // Minutes
     var appInactiveTimeout = APP_OPTIONS.inactivityTimeout; // Minutes
+    var intervalRef;
+    var isContinue = true;
```

Comment:
Clever. I wouldn't have implemented this this way, but seeing it implemented this way, this adds some elegance.